apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -40,19 +40,6 @@
 // Always include these helper files for dotcom FSE.
 require_once __DIR__ . '/dotcom-fse/helpers.php';
 
-/**
- * Load Core Site Editor.
- */
-function load_core_site_editor() {
-	require_once __DIR__ . '/site-editor/index.php';
```

Comment:
We might still need this file for enqueueing script customizations for nav sidebar, but it's probably fine to reintroduce it when the need arrises again. We could also look if it makes sense to place that code somewhere else, like in editor integration.